### DISCUSSION TITLE
Is it possible to add Vram temp sensor for fan curve?

### DISCUSSION THREAD

**Discussion by TheFuchsen** [Rem0o/FanControl.Releases#679] [Ideas]
Just since the main load my system has is when I leave it mining, would be nice to link the fan curve to the gpu memory temp instead of just the GPU core, which barely warms up.

**Comment by Fr0stX76** [comment]
Open the sensor settings from the menu (top right, 3 dots) and disable the NvApiWrapper and use the other one (GPU) provided by LibreHardwareMonitor instead.

You will have a sensor error, its normal. just refresh and new sensors should appear, including gpu mem junction

**Reply by TheFuchsen** [reply]
Had to manually relaunch the app since just refreshing did not show the other GPU sensors.

They are there now. 

![image](https://user-images.githubusercontent.com/104402512/165202618-886342dc-5fcd-45cc-80f2-aea6ce8f6d4e.png)

Thanks.
